test/functional/sqlsrv/TC24_Close.phpt

```diff
@@ -6,24 +6,22 @@ that resources are invalidated when connection is closed.
 --ENV--
 PHPT_EXEC=true
 --SKIPIF--
-<?php require('skipif.inc'); ?>
+<?php require('skipif_versions_old.inc'); ?>
 --FILE--
 <?php
 require_once('MsCommon.inc');
 
-function ConnectionClose()
+function connectionClose()
 {
     $testName = "Connection - Close";
     startTest($testName);
-
     setup();
-
-    $noRows = 5;
-    $conn1 = connect();
-    $tableName = 'TC24test';
+    $conn1 = AE\connect();
+    $tableName = 'TC24test';  
     
-    createTable($conn1, $tableName);
-    insertRows($conn1, $tableName, $noRows);
+    // Insert a couple of random rows
+    AE\createTestTable($conn1, $tableName);
+    AE\insertTestRows($conn1, $tableName, 2);
```

Comment:
why change $noRows from 5 to 2?

Reply:
Just thought the essence of the test is not really the data. Anyway, I've reverted the change.